tensorflow_datasets/core/download/downloader.py

```diff
@@ -48,6 +48,8 @@ def _get_filename(response: Response) -> str:
   content_disposition = response.headers.get('content-disposition', None)
   if content_disposition:
     match = re.findall('filename="(.+?)"', content_disposition)
+    if not match:
+        match = re.findall('filename=(.+)', content_disposition)
```

Comment:
How about only using `filename=(.+?)`, then returning `match[0].strip('"')` ?

Reply:
`filename=(.+?)` does not work at all, `filename="(.+?)"` works only with quotes, `filename=(.+) and strip will now work for certain test cases ('attachment;filename="hello.zip";''filename*=UTF-8\'\'hello.zip')

Reply:
So in short we need to check both cased. As for adding tests in `download_test.py` should I just edit the current test and add a new csid without quotes (and loop through the csids to assert if it is working)?

Reply:
For testing, I would add a parametrised test which check the various examples from: https://tools.ietf.org/html/rfc6266#section-5